.swiftlint.yml

```diff
@@ -6,6 +6,12 @@ excluded:
 opt_in_rules:
   - empty_count
   - file_header
+  - explicit_init
+  - closure_spacing
+  - overridden_super_call
+  - redundant_nil_coalescing
+  - private_outlet
+  - nimble_operator
```

Comment:
What's the rationale in enabling rules that 100% don't apply to SwiftLint?

The `nimble_operator` rule is only useful if you're using Nimble in your project...